src/commands/mods-uninstall.ts

```diff
@@ -75,13 +75,26 @@ export default new Command("mods:uninstall <modInstanceId>")
     try {
       await modsApi.deleteInstance(projectId, instanceId);
       if (confirmedServiceAccountDeletion || options.force) {
-        await iam.deleteServiceAccount(projectId, instance.serviceAccountEmail);
-        utils.logLabeledBullet(
-          logPrefix,
-          `deleted service account ${clc.bold(instance.serviceAccountEmail)}`
+        const saDeletionRes = await iam.deleteServiceAccount(
+          projectId,
+          instance.serviceAccountEmail
         );
+        if (_.get(saDeletionRes, "body.error") && _.get(saDeletionRes, "body.error.code") !== 404) {
```

Comment:
```
_.get(saDeletionRes, "body.error") && _.get(saDeletionRes, "body.error.code")
```

is redundant - see the documentation of `_.get` - only the second statement is necessary.

Reply:
I don't think it is? The first statement isn't there to null guard -  if body.error is undefined, the second statement ```_.get(saDeletionRes, "body.error.code") !== 404``` evaluates to true.

Reply:
Rewrote this to clarify whats happening